modules/internal/Bytes.chpl

```diff
@@ -585,8 +585,8 @@ module Bytes {
 
 
   /*
-    Gets a `c_ptrConst(c_char)` from a :type:`bytes`. The returned `c_ptrConst(c_char)`
-    shares the buffer with the :type:`bytes`.
+    Gets a `c_ptrConst(c_char)` from a :type:`bytes`. The returned
+    `c_ptrConst(c_char)` shares the buffer with the :type:`bytes`.
```

Comment:
I might also make this a :class:`~CTypes.c_ptrConst` reference for consistency with the other docs (even though the proc is being deprecated)